docs/zh_cn/faq.md

```diff
@@ -107,6 +107,45 @@
         ]
     ```
 
+- "How to use Mosaic augmentation"
+
+   如果你想在训练中使用 `Mosaic`，那么请确保你同时使用 `MultiImageMixDatasetresume`。以 `Faster R-CNN` 算法为例，你可以通过如下做法实现：
+   ```python
+    # 直接打开 configs/faster_rcnn/faster_rcnn_r50_fpn_1x_coco.py ,增添如下字段
+    data_root = 'data/coco/'
+    dataset_type = 'CocoDataset'
+    ​
+    img_norm_cfg = dict(
+        mean=[123.675, 116.28, 103.53], std=[58.395, 57.12, 57.375], to_rgb=True)
+
+    train_pipeline = [
+        dict(type='Mosaic', img_scale=(1333, 800), pad_val=114.0),
```

Comment:
This will be problematic, because the image will be enlarged by 4 times after Mosaic processing.